### ISSUE TITLE
Supported Desktops :: Add support for GNOME Shell 

### ISSUE THREAD

**Issue by lots0logs** [numixproject/numix-gtk-theme#389]
- [ ] Include GNOME Shell theme from downstream [Antergos/Numix-Frost](http://github.com/antergos/Numix-Frost)
- [ ] Update the GNOME Shell theme to be inline with current Numix style.

**Comment by khurshid-alam** [comment]
@lots0logs The repository of Numix Gnome Shell theme is here: https://github.com/satya164/elegance-colors

It is not maintained anymore. We need to ask @satya164 what to do with it.

**Comment by lots0logs** [comment]
@khurshid-alam The Gnome Shell theme in the Numix-Frost repo came from @satya164 :laughing: It works but it still needs to be updated at some point :grimacing:

**Comment by khurshid-alam** [comment]
@lots0logs Oh cool! So you want gnome-shell theme to be maintained here rather than at elegance-colors?

Well, I agree, I think it would be too much of a work if we are going to maintain elegance-colors (tooling, color-picker etc need rework).  The only thing we need to do is to redirect users from elegance-colors to here for reporting issues and PRs.

**Comment by satya164** [comment]
@khurshid-alam Yeah, we can maintain the Shell theme here instead of elegance colors, since it's no longer maintained. But I don't think we need to redirect users to here, since it's a completely different project.

**Comment by marmis85** [comment]
I was wondering, if you are not going to keep the elegance-colors configuration utility anyways, wouldn't it be more maintainable in the long run to rewrite the theme from scratch with SASS? The Adwaita shell theme has been written in SASS since 3.16

**Comment by satya164** [comment]
@marmis85 Wanna do it? :D

**Comment by marmis85** [comment]
@satya164  I'd love to help, but I would need some guidance, I've never really built a shell theme myself :D

**Comment by bilelmoussaoui** [comment]
I can give it a try. But never created a shell theme before. Is there any code style that is should follow ? :+1:

**Comment by sojusnik** [comment]
Did somebody found meanwhile a proper replacement for the default Adwaita gnome-shell theme for Gnome 3.18 to somehow match the Numix Gtk theme? [This](https://www.gnome-look.org/content/show.php/Numix+Style+Shell+%e2%80%93+GNOME+3.16?content=174477) one was my closest choice for 3.16, but on 3.18 it looks even worse.

**Comment by lots0logs** [comment]
@sojusnik Have you tried [NumixFrost](https://github.com/Antergos/Numix-Frost/tree/master)?

**Comment by sojusnik** [comment]
@lots0logs I know NumixFrost, but its color scheme is not matching the one in Numix. Apart from that, both the GTK and the Gnome-Shell theme look weird on Ubuntu 16.04.

**Comment by lots0logs** [comment]
@sojusnik It doesnt support ubuntu for the gtk theme. The shell theme is in need of an update for sure but it does still work for now :sweat_smile:

**Comment by lburgazzoli** [comment]
Is there any plan to release a gnome-shell theme ?
I've tried NumixFrost on Gnome 3.22 but Numix is not listed among shell themes.

**Comment by lots0logs** [comment]
@lburgazzoli How did you install it?

**Comment by lburgazzoli** [comment]
sudo make install, then tried to configure it with gnome-tweak-tool

**Comment by lots0logs** [comment]
Ah okay. The makefile doesnt install the gnome-shell theme. You will have to manually copy it into the theme's directory (`/usr/share/themes/Numix-Frost`)

**Comment by lburgazzoli** [comment]
I see, thx.

btw the theme appear to be a little bit too space consuming:
- numix:
  ![screenshot from 2016-11-03 09-45-09](https://cloud.githubusercontent.com/assets/1868933/19960131/228dded2-a1ab-11e6-872a-2a687d4678c7.png)
- arc:
  ![screenshot from 2016-11-03 09-49-24](https://cloud.githubusercontent.com/assets/1868933/19960130/228cb764-a1ab-11e6-94f5-c3cd125b8ddf.png)

Is there any plan to strip this wasting space ?

**Comment by lots0logs** [comment]
Yes there will be some adjustments. Though its not going to be as much as the regular numix theme which IMO doesnt have enough space in key areas.

**Comment by lburgazzoli** [comment]
I do not know which areas you are thinking about but I think window borders and tabs should use less space as possible, i.e. when a window is maximized you do not want the top-bar, then the window border and maybe some tabs to waste precious space especially while coding :)

Maybe you could create a Numix-Slim variant which uses less space as possible or make it easy to configure it (do not know how feasible is that).

**Comment by khurshid-alam** [comment]
Downstream issue.

**Comment by sojusnik** [comment]
Any updates on the Gnome shell theme? [Gnome 3.24](https://www.gnome.org/news/2017/03/gnome-3-24-released/) is already out.
Since Gnome will soon become the default DE in Ubuntu again, it would be the right moment to work on a Numix Gnome shell theme.

**Comment by sojusnik** [comment]
It seems, that somebody recently started to work on that: https://github.com/hsharrison/numix-gnome-theme

**Comment by Corsseir** [comment]
Hey, I can start working on theme immediately, but I would like to ask you to create subdirectory of numixproject called for example numix-gnome-shell-theme. I will commit my changes there. I have 3+ years experience with html5, css3 and javascript(jquery, angular, react) and I work as frontend developer.

**Comment by lots0logs** [comment]
The theme would be maintained inside this repo actually. You can submit a PR with the theme located here:  `src/desktops/gnome-shell`

**Comment by Corsseir** [comment]
Ok, thanks for response.

**Comment by khurshid-alam** [comment]
@Corsseir We already have it with somewhat a different name : https://github.com/numixproject/elegance-colors

@lots0logs Wouldn't it be better if this goes into numixproject/numix-gnome-shell-theme ?

**Comment by lots0logs** [comment]
@khurshid-alam I don't think so. Remember we had discussed this on Telegram a few months ago?

**Comment by khurshid-alam** [comment]
@Corsseir https://github.com/numixproject/numix-gnome-shell-theme

**Comment by lots0logs** [comment]
@Corsseir As you can see, we haven't come to an agreement internally as to how we are going to structure things going forward. But don't worry, that won't affect any work you contribute to the gnome-shell theme. It'll be fine no matter where we end up putting the files :wink:

**Comment by Corsseir** [comment]
Hah, ok, no problem for me :)

**Comment by khurshid-alam** [comment]
Yes, we came to an agreement. For the time being we will use https://github.com/numixproject/numix-gnome-shell-theme.

**Comment by sojusnik** [comment]
Is there any progress on the gnome shell theme?

**Comment by Corsseir** [comment]
yep, stay tuned i will push it soon.

**Comment by ghost** [comment]
Hello, any update on this?
Thanks!

**Comment by paspro** [comment]
Perhaps next year... In the meantime I use the attached theme I adjusted for GNOME 3.26 and its support for transparencies which works great with the Numix GTK theme. One may want to adjust the font to use in gnome-shell.css as well as the base theme:

```
@import url("/usr/share/gnome-shell/theme/ubuntu.css");

stage {
	font-family: Roboto Regular;
	font-size: 11pt;
}

```

[Numix-Gnome-Shell-Theme.zip](https://github.com/numixproject/numix-gtk-theme/files/1460824/Numix-Gnome-Shell-Theme.zip)
